client/flutter/lib/api/content/content_loading.dart

```diff
@@ -24,6 +29,18 @@ class ContentLoading {
 
   ContentLoading._internal();
 
+  /// Pre-cache commonly used content bundles to ameliorate loading delay.
+  /// This method may be called at any time and will only load content when needed.
+  void preCacheContent(Locale locale) {
+    print("preCachecontent for locale: ${locale}");
+    AdviceContent.travelAdvice(locale);
+    FactContent.getTheFacts(locale);
+    IndexContent.homeIndex(locale);
+    IndexContent.learnIndex(locale);
+    QuestionContent.yourQuestionsAnswered(locale);
+    SymptomCheckerContent.load(locale);
```

Comment:
@brunobowden I did think about this :)  So with Dart capabilities at the moment something has to know about all of the content schema classes to either trigger the initialization, registration, or reflection on them.  (In some languages there are ways to search for annotations on classes, etc. but not in Dart yet, I believe).  

What we would have to do is have either a tiny build time script that generates a loader based on the files in the `content/schema` directory or at runtime try to infer the classes by looking the content bundle names.  I decided that a) this may be a bit of overkill at the moment since there are so few files and b) we may actually want to make a manual decision about which ones to load and when... e.g. stage them, etc. at least in the short term...  We could probably do the latter with annotations on the classes later.  Probably best to see how this goes for a bit.